modules/install/rke2_master.sh

```diff
@@ -1,121 +1,151 @@
 #!/bin/bash
 # This script installs the first master, ensuring first master is installed
 # and ready before proceeding to install other nodes
-#!/bin/bash
-set -x
-echo "$@"
-
-node_os=$1
-create_lb=$2
-rke2_version=$3
-public_ip=$4
-rke2_channel=$5
-server_flags=$6
-install_mode=$7
-rhel_username=$8
-rhel_password=$9
-install_method=${10}
-
-hostname=$(hostname -f)
+
+create_directories() {
 mkdir -p /etc/rancher/rke2
-cat << EOF >/etc/rancher/rke2/config.yaml
+}
+
+create_config() {
+  local fake_fqdn="${1}"
```

Comment:
if using locals, give a break line to rest of the code, to make it more readable